packages/shared/src/components/squads/SquadHeaderBar.tsx

```diff
@@ -55,43 +53,34 @@ export function SquadHeaderBar({
     <div
       {...props}
       className={classNames(
-        'flex h-fit w-full flex-row justify-center gap-4 tablet:w-auto',
+        'no-scrollbar flex h-fit w-full flex-row justify-center gap-4 overflow-x-auto pr-4 tablet:w-auto tablet:pr-0',
         className,
       )}
     >
-      <div className="relative">
-        {verifyPermission(squad, SourcePermissions.Invite) &&
-          !showJoinButton && (
-            <Button
-              variant={ButtonVariant.Secondary}
-              className={classNames(
-                tourIndex === TourScreenIndex.CopyInvitation &&
-                  'highlight-pulse',
-              )}
-              onClick={() => {
-                trackAndCopyLink();
-              }}
-              icon={<AddUserIcon />}
-              disabled={copying}
-            >
-              Invitation link
-            </Button>
+      {verifyPermission(squad, SourcePermissions.Invite) && !showJoinButton && (
```

Comment:
Nothing really changed here part from removing the extra wrapper and `ml-14 tablet:ml-0` class.